mmpose/models/keypoint_heads/top_down_simple_head.py

```diff
@@ -104,7 +104,7 @@ def __init__(self,
                             out_channels=conv_channels,
                             kernel_size=num_conv_kernels[i],
                             stride=1,
-                            padding=padding))
+                            padding=1 if num_conv_kernels[i] == 3 else 0))
```

Comment:
What's the philosophy of choosing padding? Should it be `(kernelsize - 1) // 2`?